API-Documentation.md

```diff
@@ -0,0 +1,415 @@
+| `zip`                    | `string` | Shopper's zip                                                                      | no       |
+| `email`                  | `string` | Shopper email                                                                      | no       |
+| `walletAddress`          | `string` | Shopper wallet address                                                             | no       |
+| `vatId`                  | `string` | The shopper’s VAT number                                                           | yes      |
+| `taxId`                  | `string` | The shopper’s TAX ID number                                                        | yes      |
+| `maliciousAttempts`      | `integer`| The number of malicious attemps for a shopper. It is incremented when a given signed transaction reverts. | no |
+
+### Payment
+
+| Attribute            | Type     | Description                                                                     | Nullable |
+| -------------------- | -------- | ------------------------------------------------------------------------------- | -------- |
+| `_id`                | `string` | The id of the payment                                                           | no       |
+| `status`             | `string` | The status of the Payment                                                       | no       |
+| `date`               | `date`   | The date time when the payment was created                                      | no       |
+| `currency`           | `string` | Currency code (ISO 4217) of the amount to be charged                            | no       |
+| `shopper`            | `string` | The shopperID of the shopper that will be charged                               | no       |
+| `vendor`             | `string` | The vendorID of the vendor that will charge the payment                         | no       |
+| `items`              | `array`  | Objects containing information about the items that are being purchased         | no       |
+| `fundTxData`         | `object` | Object containing information about the funding of an shopper                   | no       |
+| `genericTransactions`| `array`  | Objects containing information about every transaction that should be executed  | no       |
+| `totalAmount`        | `decimal` | The total amount of the payment, represented in the selected currency          | no       |
+
+
+### Item
+
+| Attribute            | Type     | Description                                                                     | Nullable |
+| -------------------- | -------- | ------------------------------------------------------------------------------- | -------- |
+| `description`        | `string` | Description of the item that is being bought. This information is displayed in the invoice. | no       |
+| `amount`             | `decimal`| Fiat amount of the item                                                         | no       |
```

Comment:
rename it to be `lineAmount`

Reply:
Also we have to tell that the amount is the total amount of the line without VAT